server/api/checker.go

```diff
@@ -24,13 +24,13 @@ import (
 )
 
 type checkerHandler struct {
-	*server.Handler
-	r *render.Render
+	handler *server.Handler
```

Comment:
Why change it?

Reply:
`server.Handler` holds the same-name function, so needs to change it to a named field